scripts/spi-e2e-setup.sh

```diff
@@ -9,7 +9,7 @@ set -u
 echo '[INFO] Deploying SPI OAuth2 config'
 
 export OAUTH_URL='spi-oauth-route-spi-system.'$( oc get ingresses.config/cluster -o jsonpath={.spec.domain})
-export tmpfile=tmp/config.yaml
+export tmpfile=$(mktemp -d)/config.yaml
```

Comment:
[running the spi-e2e-setup.sh script from infra-deployments CI job](https://github.com/redhat-appstudio/infra-deployments/pull/402/files) returns this error:
```bash
[INFO] Deploying SPI OAuth2 config
Please go to https://github.com/settings/developers.
And register new Github OAuth application for callback https://spi-oauth-route-spi-system.apps.ci-op-5hcgyybj-7da12.origin-ci-int-aws.dev.rhcloud.com/github/callback
bash: line 31: tmp/config.yaml: No such file or directory
```